TimberAPI/AssetLoaderSystem/ResourceAssetPatch/ResourceAssetLoaderTimberApi.cs

```diff
@@ -0,0 +1,80 @@
+using Bindito.Core;
+using HarmonyLib;
+using Timberborn.AssetSystem;
+using TimberbornAPI.AssetLoaderSystem.AssetSystem;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace TimberbornAPI.AssetLoaderSystem.ResourceAssetPatch
+{
+    public class ResourceAssetLoaderTimberApi : IResourceAssetLoader
```

Comment:
I feel like a better name is "TimberApiResourceAssetLoader" instead of "ResourceAssetLoaderTimberApi"